src/Components/PostInfo.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Props, UserInfo } from './UserInfo';
```

Comment:
```suggestion
import { UserInfo } from './UserInfo';

type Props = {
```
Краще явно прописувати тип для пропсів прямо тут, а не імпортувати